### PR TITLE
Take my changes

### PR DESCRIPTION
**Pull request by oxivanisher**
:dagger:

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+MQTT_HOST="localhost"
+
+INFLUX_HOST="localhost"
+INFLUX_PORT=8086
+INFLUX_USER="username"
+INFLUX_PASS="password"
+INFLUX_DABA="database"
+
+#InfluxDB container config
+INFLUXDB_HTTP_BIND_ADDRESS=":${INFLUXDB_PORT}"
```
Comment (fujexo) [inline .env]: INFLUXDB_PORT doesn't exist, should be INFLUX_PORT I think

File: influxdb/etc/influxdb.conf [hunk 2]
```diff
@@ -1,7 +0,0 @@
-[meta]
-  dir = "/var/lib/influxdb/meta"
```
Comment (fujexo) [inline influxdb/etc/influxdb.conf]: I would leave this out, these are defaults which work great within docker. Just have to map volumes correctly though

File: mqtttransport/mqtt-influx.py [hunk 3]
```diff
@@ -2,22 +2,23 @@
...
+influx_port = os.environ['INFLUX_PORT']
+influx_user = os.environ['INFLUX_USER']
+influx_pass = os.environ['INFLUX_PASS']
+influx_daba = os.environ['INFLUX_DABA']
 
 # MQTT settings
-mqtt_host = "localhost"
-mqtt_port = 1883
+mqtt_host = os.environ['MQTT_HOST']
+mqtt_port = os.environ['MQTT_PORT']
```
Comment (fujexo) [inline mqtttransport/mqtt-influx.py:18]: Must be cast into an integer like ```int(var)```. paho-mqtt doesn't like string for a port

File: mqtttransport/mqtt-influx.py [hunk 4]
```diff
@@ -2,22 +2,23 @@
...
 
 # Logging settings
-logging.basicConfig(format='%(asctime)s %(message)s', level=logging.INFO)
+logging.basicConfig(format='%(asctime)s %(message)s', level=logging.DEBUG)
```
Comment (fujexo) [inline mqtttransport/mqtt-influx.py:21]: Why DEBUG?

Comment (oxivanisher) [reply mqtttransport/mqtt-influx.py:21]: because you said so ^^

Comment (fujexo) [reply mqtttransport/mqtt-influx.py:21]: but only for debugging purposes :D

**Comment by fujexo** [comment]
Please fix the things I commented, after that I can merge that good stuff :)

**Comment by oxivanisher** [comment]
something is broken. you will get a new pull request

**Pull request closed without merging**